web/private/scripts/drush_config_import.php

```diff
@@ -1,9 +1,13 @@
 <?php
```

Comment:
Should we rename this file to drush_post_deploy.php?

Reply:
This would require an update to pantheon.yml as well, but the note about what the script does there should be updated anyway.

Reply:
Makes sense. Made the update